app/assets/stylesheets/darkswarm/menu.css.scss

```diff
@@ -1,8 +1,8 @@
-@import "compass";
-@import "branding";
-@import "mixins";
-@import "typography";
-@import "variables";
```

Comment:
we are using double quotes in all css files right now, I'd keep the local dialect.

Reply:
Yes, glad to be back as I've finally graduated and found a job 😄 Sorry about the quotation marks - it was done by my Prettier, I'll remember to disable it next time!

Reply:
new job, congrats!